vue-frontend/src/components/contact/NewContactForm.vue

```diff
@@ -0,0 +1,567 @@
+      calendarIcon: faCalendarAlt,
+      faCaretDown,
+      faCaretUp,
+      showList: false,
+      floatable: false,
+      name: "",
+      email: "",
+      projectInfo: "",
+      reference: "",
+      invest: "",
+      investOptions: [
+        "< USD 50000",
+        "USD 50000 - USD 100000",
+        "USD 100000 - USD 250000",
+        "> USD 250000",
+      ],
+      NDA: false,
+      showNameValidationError: false,
+      showEmailValidationError: false,
+      showValidEmailError: false,
+      showProjectInfoValidationError: false,
+      showReferenceValidationError: false,
+      showInvestValidationError: false,
+      openCalendlyIframeModal: false,
+      errorMessage: "Something went wrong on our side",
+      showLoader: false,
+      showSuccessMessage: false,
+      showErrorMessage: false,
+      contactType: 1,
+    };
+  },
+  components: {
+    FontAwesomeIcon,
+    CalendlyIframe,
+  },
+  inject: ["mixpanel"],
+  mounted() {
+    document.addEventListener("click", this.closePopUps);
+  },
+  unmounted() {
+    document.removeEventListener("click", this.closePopUps);
+  },
+  methods: {
+    isValidEmail() {
+      var emailRegx =
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+      return !emailRegx.test(this.email);
+    },
+    validateForm() {
+      this.showNameValidationError = this.name.trim().length === 0;
+      this.showEmailValidationError = this.email.trim().length === 0;
+      this.showProjectInfoValidationError =
+        this.projectInfo.trim().length === 0;
+      this.showReferenceValidationError = this.reference.trim().length === 0;
+      this.showInvestValidationError = this.invest.trim().length === 0;
+      return (
+        this.showNameValidationError ||
+        this.showEmailValidationError ||
+        this.showValidEmailError ||
+        this.showProjectInfoValidationError ||
+        this.showReferenceValidationError ||
+        this.showInvestValidationError
+      );
+    },
+    submitForm() {
+      if (!this.validateForm()) {
+        this.mixpanel.track("tap_contact_submit_button");
+        let formData = {
+          name: this.name,
+          email: this.email,
+          project_info: this.projectInfo ? this.projectInfo : "NA",
+          reference: this.reference,
+          invest: this.invest,
+          nda: this.NDA,
+          contact_type:
+            this.contactType == CONTACT_BY_CHAT_OR_MAIL
+              ? "Chat or Email"
+              : "Call",
+        };
+
+        grecaptcha.enterprise.ready(() => {
+          grecaptcha.enterprise
+            .execute(import.meta.env.VITE_RECAPTCHA_SITE_KEY, {
+              action: "verify",
+            })
+            .then((token) => {
+              formData.token = token;
+              axios
+                .post(config.API_BASE + "/api/send-contact-mail", formData)
+                .then(() => {
```

Comment:
Can't we call `this.resetForm();` after then() instead of putting it in both if and else loop?